### PR TITLE
IGAPP-459: Black line shown on not found page

### PR DESCRIPTION
**Pull request by maxammann**
I decided to replace the module with a custom implementation as our usecase is quite simple.
Tested and works on IE11. RTL/LTR support is not working on IE, but I think this is not a huge issue.

### CODE REVIEW

File: web/src/modules/common/components/Tooltip.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
+// @flow
+
+import * as React from 'react'
+import styled from 'styled-components'
+import dimensions from '../../theme/constants/dimensions'
+
+const renderPseudoClasses = (direction: string) => `
```
Comment (maxammann) [inline web/src/modules/common/components/Tooltip.js:7]: Is there an other way of composing styled components?

File: web/src/modules/common/components/Tooltip.js [hunk 2]
```diff
@@ -0,0 +1,180 @@
+// @flow
```
Comment (SteffiStuffel) [inline web/src/modules/common/components/Tooltip.js:1]: Why did decide to add is at standalone instead of using yarn package?

**Review by SteffiStuffel — APPROVED** [review]
Tested this, no black line is shown.

Have reviewed the code but cannot say anything about the code.

**Review by klinzo — APPROVED** [review]
Well done :+1:

File: web/src/modules/common/components/Tooltip.js [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+      border-bottom-color: #333;
+    `}
+    ${flow === 'left' && `
+      ${toLogicalProperty('border-right-width', supportsLogicalProperties)}: 0;
+      ${toLogicalProperty('border-left-color', supportsLogicalProperties)}: #333;
+      ${toLogicalProperty('left', supportsLogicalProperties)}: -5px;
+    `}
+    ${flow === 'right' && `
+      ${toLogicalProperty('border-left-width', supportsLogicalProperties)}: 0;
+      ${toLogicalProperty('border-right-color', supportsLogicalProperties)}: #333 ;
```
Comment (Schedulaar) [inline web/src/modules/common/components/Tooltip.js:56]:
```suggestion
      ${toLogicalProperty('border-right-color', supportsLogicalProperties)}: #333;
```

File: web/src/modules/common/components/Tooltip.js (hunk 2 above)
Comment (maxammann) [reply web/src/modules/common/components/Tooltip.js:1]: This is my implementation :)

File: web/src/modules/layout/components/Header.js [hunk 4]
```diff
@@ -30,6 +30,7 @@ const HeaderContainer = styled.header`
   background-color: ${props => props.theme.colors.backgroundAccentColor};
   user-select: none;
   flex-direction: column;
+  overflow: visible;
```
[the remark this answers is not in the document]
Comment (maxammann) [reply web/src/modules/layout/components/Header.js:33]: Else a scrollbar appears when the tip opens.

File: web/src/modules/feedback/components/FeedbackToolbarItem.js [hunk 5]
```diff
@@ -36,15 +31,14 @@ export class FeedbackToolbarItem extends React.PureComponent<PropsType> {
...
-          data-event-off='click mouseout'
-          icon={isPositiveRatingLink ? faSmile : faFrown} />
-        {viewportSmall && <StyledSmallViewTip>{smallViewTip}</StyledSmallViewTip>}
-      </StyledFeedbackToolbarItem>
+      <Tooltip text={dataTip} flow='up' mediumViewportFlow='right' smallViewportFlow='down'>
+        <StyledFeedbackToolbarItem className={className} onClick={this.handleLinkClick} aria-label={dataTip}>
+          <FontAwesomeIcon
+            className={className}
+            icon={isPositiveRatingLink ? faSmile : faFrown} />
+          {viewportSmall && <StyledSmallViewTip>{smallViewTip}</StyledSmallViewTip>}
```
[the remark this answers is not in the document]
Comment (maxammann) [reply web/src/modules/feedback/components/FeedbackToolbarItem.js:39]: Not really but I think this still looks quite good

[the remark this answers is not in the document]
Comment (klinzo) [reply web/src/modules/feedback/components/FeedbackToolbarItem.js:39]: What does it do?

[the remark this answers is not in the document]
Comment (maxammann) [reply web/src/modules/feedback/components/FeedbackToolbarItem.js:39]: I removed tooltips for small viewport

**Review by Taggotty — APPROVED** [review]

**Pull request merged**